website/docs/r/default_vpc.html.markdown

```diff
@@ -15,10 +15,6 @@ For AWS accounts created after 2013-12-04, each region comes with a Default VPC.
 **This is an advanced resource**, and has special caveats to be aware of when
 using it. Please read this document in its entirety before using this resource.
 
-The `aws_default_vpc` behaves differently from normal resources, in that
```

Comment:
I believe it's worth adding a note to say that it now creates it if it didn't exist prior

Reply:
Changed as suggested.